dts/bindings/qspi/nxp,s32-qspi-sfp-frad.yaml

```diff
@@ -0,0 +1,62 @@
+# Copyright 2024 NXP
+# SPDX-License-Identifier: Apache-2.0
+
+description: NXP S32 Quad Serial Peripheral Interface (QSPI) Secure Flash Protection SFP FRAD.
+
+  The SFP FRAD performs second-level checks on input flash write and erase transactions,
+  based on the address range of each transaction.
+
+compatible: "nxp,s32-qspi-sfp-frad"
+
+include: base.yaml
+
+properties:
+  start-address:
+    type: int
+    required: true
+    description: |
+      Specifies the specific flash memory region start address.
+
+  end-address:
+    type: int
+    required: true
+    description: |
+      Specifies the specific flash memory region end address.
+
+  master-domain-acp:
+    type: string
+    enum:
+      - ACP NONE
+      - ACP SECURE
+      - ACP SECURE PRIVILEGED
+      - ACP ALL
+      - ACP PRIVILEGED
```

Comment:
we can drop the ACP prefix in all enums, already mentioned in the property

Reply:
I droped